core/src/main/scala/org/apache/spark/sql/row/JDBCMutableRelation.scala

```diff
@@ -367,6 +368,38 @@ case class JDBCMutableRelation(
       ifExists: Boolean): Unit = {
     throw new UnsupportedOperationException()
   }
+
+  override def alterTable(tableIdent: QualifiedTableName,
```

Comment:
Might have conflict with update/delete work on column table. Please check Sumedh's branch for his update/delete work so both of you don't have to resolve conflicts. Just a suggestion.